src/DevBetterWeb.Core/Services/NewMemberService.cs

```diff
@@ -125,17 +127,6 @@ private Task AddUserToMemberRoleAsync(string userId)
       return _userRoleMembershipService.AddUserToRoleByRoleNameAsync(userId, roleName);
     }
 
-    private async Task CreateSubscriptionForMemberAsync(int memberId, DateTimeRange subscriptionDateTimeRange)
-    {
-      var subscription = new Subscription();
-      subscription.MemberId = memberId;
-      subscription.Dates = subscriptionDateTimeRange;
-
-      var member = await _memberRepository.GetByIdAsync(memberId);
-      if (member is null) throw new MemberNotFoundException(memberId);
-      member.AddSubscription(subscription);
-    }
-
```

Comment:
I see this is the original code the service is replacing. I'm not sure what code calls this, but again if you shrink down subscription (use a constructor) and get the member elsewhere if needed, this code should shrink to almost nothing.